app/hotels/src/singleHotel/roomList/RoomRow.js

```diff
@@ -29,36 +30,49 @@ type Props = {|
   +testID?: string,
   +availableRoom: ?RoomType,
   +navigation: NavigationType,
-  +select: (availabilityId: string, maxPersons: number) => void,
-  +deselect: (availabilityId: string, maxPersons: number) => void,
-  +selected: {
-    +[string]: number,
-  },
-  +numberOfRooms: number,
   +getGuestCount: () => number,
+  +numberOfRooms: number,
+  +relay: RelayProp,
 |};
 
 export class RoomRow extends React.Component<Props> {
-  isDisabled = () => this.props.getGuestCount() <= this.props.numberOfRooms;
+  isDisabled = () => this.props.numberOfRooms >= this.props.getGuestCount();
```

Comment:
This change does not seem necessary?

Reply:
(But I'm fine with it 😉 )

Reply:
It is more readable in my opinion, that's why I changed it 😊